tests/test_matrix_utils_auth.py

```
@@ -0,0 +1,1264 @@
+        mock_client_instance.rooms = {}
+
+        async def mock_whoami():
+            """
+            Create a fake whoami response for tests.
+            
+            Returns:
+                MagicMock: A mock object with a `device_id` attribute set to `"test_device_id"`.
+            """
+            return MagicMock(device_id="test_device_id")
+
+        async def mock_sync(*args, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `args`

(ARG001)